docs/cookbook/domain/add_command.md

```diff
@@ -0,0 +1,84 @@
+# How create custom command
+
+## Creating domain command
+
+Domain command should represent single transactional business logic.  
```

Comment:
Brakuje s dla represents: Domain command should represents single transactional business logic.